src/components/routes/qdm/TestCaseRoutes.test.tsx

```diff
@@ -149,8 +150,11 @@ describe("TestCaseRoutes", () => {
     expect(tBody.rows.item(0).cells[1]).toHaveTextContent("IPP_Pass");
     expect(tBody.rows.item(0).cells[2]).toHaveTextContent("TC12");
     expect(tBody.rows.item(0).cells[3]).toHaveTextContent("Desc1");
+    expect(tBody.rows.item(0).cells[4]).toHaveTextContent(
+      "2024-09-10 08:40:14.382"
```

Comment:
I think we should confirm the date format and the display level once with PO/UI-UX

Reply:
@adongare Ok sure will do.

Reply:
@adongare This is changed to display only date